hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/exec/scope/HandleHederaOperations.java

```diff
@@ -481,8 +471,7 @@ private ContractCreateTransactionBody standardized(
         }
     }
 
-    private boolean needsStandardization(
-            @NonNull final ContractCreateTransactionBody op, @Nullable final Key standardAdminKey) {
-        return op.hasInitcode() || op.gas() > 0L || op.initialBalance() > 0L || standardAdminKey != op.adminKey();
+    private boolean needsStandardization(@NonNull final ContractCreateTransactionBody op) {
```

Comment:
Now that the argument `standardAdminKey` is gone - isn't clear to me what `needsStandardization` means.  Perhaps change the name of the method or add javadoc?

Reply:
added javadoc